doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -20,6 +20,7 @@ To get started with your nRF52 Series DK, follow the steps in the :ref:`ug_nrf52
 If you are not familiar with the |NCS| and its development environment, see :ref:`installation` and :ref:`configuration_and_build` documentation.
 
 .. _ug_nrf52_developing_ble_fota:
+
 .. fota_upgrades_intro_start
 
 FOTA updates
```

Comment:
I suggest that we link to [DFU docs](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-14939/nrf/config_and_build/dfu/index.html) from just after |fota_upgrades_def|.
Something like 
"You can also use FOTA updates to replace the application. See ref:dfu for general DFU and FOTA documentation."

Reply:
And the same for FOTA Updates supersection for nRF5340.

Reply:
Good point - I will give this a think and implement something, would just need to think of how to describe the context around the link to DFU page (since this PR adds link to FOTA pages on the DFU one, so user isn't just going back and forth). Please let me know if you have any suggestions in the meantime, or after I implement

Reply:
I see that you added a sentence on the nRF52840 FOTA docs, nice!
Did you not put the same under nRF5340 FOTA docs on purpose?

Reply:
The nRF5340 FOTA section is copying the text from the nRF52 Series page  - see doc build [preview](https://nordicsemi.atlassian.net/browse/NCSIDB-1158) , let me know if I have missed something

Reply:
Aha, I see